test/TestProjects/Accessibility-LowLevel-NoAuth/src/Generated/Configuration.json

```diff
@@ -7,5 +7,6 @@
     "..\\..\\..\\..\\..\\artifacts\\bin\\AutoRest.CSharp\\Debug\\net7.0\\Azure.Core.Shared"
   ],
   "public-clients": true,
-  "skip-csproj-packagereference": true
+  "skip-csproj-packagereference": true,
+  "generate-test-project": true
```

Comment:
Can we enable some projects with  "generate-sample-project": true

Reply:
its default value is `true` therefore we always have sample projects.